nautobot/core/templates/inc/object_details_advanced_panel.html

```diff
@@ -54,6 +56,16 @@
                         <td>{{ object.last_updated }}</td>
                     </tr>
                 {% endif %}
+                {% if object|validated_api_viewname:'detail' %}
+                    <tr>
+                        <td>View in API Browser</td>
+                        <td>
+                            <a href="{% url object|api_viewname:'detail' object.id %}" target="_blank">
```

Comment:
Can we store and reuse the viewname returned from `validated_api_viewname` above rather than re-looking it up a second time?